### PR TITLE
Migrate guide from old site

### PR DESCRIPTION
**Pull request by justanwar**

### CODE REVIEW

File: guide/reindex-wallet.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+author: "Anwar P."
+permalink: /guide/reindex-wallet.html
+---
+## Reindexing your wallet
+
+If you have trouble syncing your wallet and it appears to be stuck on a block, please start it again with the **-reindex-chainstate flag**. This will start the Zcoin wallet and begin the blockchain reindex process and **will take several hours**. 
+
+If you suspect corrupted blocks, please run **\-reindex** instead. 
+
+Make sure you are always on the latest release by going to our [release page](https://github.com/zcoinofficial/zcoin/releases).
```
Comment (ajaydono) [inline guide/reindex-wallet.md:15]: https://github.com/firoorg/firo/releases

Comment (ajaydono) [inline guide/reindex-wallet.md:15]: Make sure you are always on the latest release by going to our [release page](https://github.com/firoorg/firo/releases).

**Review by ajaydono — CHANGES_REQUESTED** [review]
https://github.com/firoorg/firo/releases

File: guide/reindex-wallet.md (hunk 1 above)
Comment (justanwar) [reply guide/reindex-wallet.md]: Fixed

**Pull request merged**